testsuite/features/support/env.rb

```diff
@@ -58,7 +58,7 @@
 $is_using_build_image = ENV.fetch('IS_USING_BUILD_IMAGE', false)
 $is_using_scc_repositories = (ENV.fetch('IS_USING_SCC_REPOSITORIES', 'False') != 'False')
 $catch_timeout_message = (ENV.fetch('CATCH_TIMEOUT_MESSAGE', 'False') == 'True')
-$beta_enabled = ENV.fetch('BETA_ENABLED', false)
+$beta_enabled = (ENV.fetch('BETA_ENABLED', 'False') == 'True')
```

Comment:
would be better to use "yes" like for MIRROR, but ok

Reply:
For now, let's just fix this.